packages/state-transition/src/util/electra.ts

```diff
@@ -20,7 +20,11 @@ export function switchToCompoundingValidator(state: CachedBeaconStateElectra, in
   // directly modifying the byte leads to ssz missing the modification resulting into
   // wrong root compute, although slicing can be avoided but anyway this is not going
   // to be a hot path so its better to clean slice and avoid side effects
-  const newWithdrawalCredentials = validator.withdrawalCredentials.slice();
+  const newWithdrawalCredentials = Uint8Array.prototype.slice.call(
```

Comment:
was checking where we do `Uint8Array.prototype.slice` currently but it does not seem to be used anywhere besides https://github.com/ChainSafe/lodestar/blob/b785bbdedfc90c8d586a79601cfa4a4f40175cd8/packages/cli/src/cmds/validator/keymanager/keystoreCache.ts#L41

Reply:
we use it a bunch in ssz library though